### ISSUE TITLE
Paladin has less HP than Crusader (with Composite Armor upgrade)

### ISSUE THREAD

**Issue by ExiLe-ZH** [TheSuperHackers/GeneralsGamePatch#409]
Now the whole composite armor subject is quite controversial topic but here are the options possible:

```
1. Keep as is (and fix code), only crusader benefits from double armor buff. 	-> Crusader 680HP, Paladin 600HP
2. Remove bug, both get 20% armor bonus. 					-> Crusader 580HP, Paladin 600HP
3. Give same 40% buff to Paladin (potentially makes Paladin OP). 		-> Crusader 680HP, Paladin 700HP
4. Give both 30%. 								-> Crusader 625HP, Paladin 650HP
5. Give Paladin the same total HP (+180HP). 					-> Both 680HP
```

**Comment by ExiLe-ZH** [comment]
Currently option 1 is applied right?

Imo it still doesn't make sense, personally I prefer option 5.

**Comment by commy2** [comment]
Correction: Crusader (and Microwave) start with 480 HP, while Paladin starts with 500 HP.

**Comment by ExiLe-ZH** [comment]
> 
> 
> Correction: Crusader (and Microwave) start with 480 HP, while Paladin starts with 500 HP.

So what's incorrect?

**Comment by Jundiyy** [comment]
I'd prefer to see the Paladin buffed but don't want to say my vote on that due to fear of being op.
Crusader shouldn't change in my opinion, it's known as a strong unit and used for that reason.

**Comment by xezon** [comment]
To get a more complete picture, it would be nice to see more stats of Crusader and Paladin. For example damage and rate of fire. We need a spreadsheet of all units.

**Comment by commy2** [comment]
Damage, range and rate of fire are identical for both Crusader and Paladin.

The only difference between both weapons is, that the Crusaders tank shell flies at 400 dst/s, while the Paladins tank shell only flies at 300 dst/s.

At max range of 150 dst, the Paladins tank shell takes .5 seconds to arrive at the target, while the Crusaders tank shell arrives after only .375 seconds. Since the game runs at 30 FPS, the Paladin tank shells travels for 15 frames, while the Crusader tank shell travels for 12 frames.

These 3 frames _could_ come in clutch every second full moon or so and grant you a kill before the enemy can fire back.

The sad fact is that the Paladins gun is ever so slightly worse than the Crusaders gun.

**Comment by Jundiyy** [comment]
They are literally the same Tank, although imagine they hadn't removed the Crusader anti inf weapon... (still can be seen and used with a removal of the ;) it would be an even better unit, taking down infantry with the CA without a problem.

**Comment by xezon** [comment]
Ok if the Paladin Tank is worse than Crusader Tank, then it is reasonable to do something about it. In all game types Paladin Tank is almost never used.

**Comment by commy2** [comment]
The Paladin tank is better than a Crusader in that it starts with 20 extra HP (+4.2%, not much) and a slow firing PDL that can target infantry, especially GLA Workers when dropped by Chinook into a boxed in Supply Stash.

In general, it is not worth the extra $200 and 2 second build-time. After Composite Armor, it has much less HP than a Crusader to the point where the Crusader survives longer against 2 rocket infantries than a Paladin does despite the PDL.

1.04
Crusader $900 13.25 seconds
Paladin $1100 15.25 seconds

**Comment by xezon** [comment]
Paladin Tank also costs a General Promotion point which I think is of major consideration here.

**Comment by EnigmaZift** [comment]
Just fix the Composite Armor bug (it should be just once not twice). This is exactly what was intended by the original devs of the game and what the game was probably balanced for originally.

So basically Number 2

**Comment by MTKing4** [comment]
option 2. remove bug 
crusader is OP as it is, if it turns out too weak vs chinas we can then revert some of the unit price/cost changes that 1.04+ has

**Comment by ReLaX82** [comment]
Make HP of Paladin better than crusader with comp armour by a small, reason able amount of 10 to 20%.

Paladin Costs a gen point as well.

**Comment by xezon** [comment]
Fixed with #777
